### PR TITLE
[C# sample] 02.messageExtensions.a.searchCommand

### PR DESCRIPTION
**Pull request by swatDong**
Part of https://github.com/microsoft/teams-ai/issues/292, to port the basic search command message extension sample.
A bit different from the js sample - in this .net one, search for NuGet package instead of NPM package.
Verified on both local and azure.

### CODE REVIEW

**Review by kuojianlu — APPROVED** [review]

File: dotnet/samples/02.messageExtensions.a.searchCommand/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This sample shows how to incorporate a basic Message Extension app into a Microsoft Teams application using [Bot Framework](https://dev.botframework.com) and the Teams AI SDK. Users can search nuget.org for packages.
+
+## Set up instructions
+
+All the samples in for the C# .NET SDK can be set up in the same way. You can find the step by step instructions here:
+ [Setup Instructions](../README.md).
+
+## Interacting with the Message Extension
+
+You can interact with this app by selecting it's app icon in the chat compose area. This opens a dialog that allows you to search NuGet for a package. Selecting a package will output an Adaptive Card with it's description to the chat.
```
Comment (kuojianlu) [inline dotnet/samples/02.messageExtensions.a.searchCommand/README.md:12]: it's -> its?

Comment (swatDong) [reply dotnet/samples/02.messageExtensions.a.searchCommand/README.md]: fixed

File: dotnet/samples/02.messageExtensions.a.searchCommand/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
```
Comment (singhk97) [inline dotnet/samples/02.messageExtensions.a.searchCommand/appsettings.json:5]: Let's change this to "Information" to enable logging in the OpenAI Planner

File: dotnet/samples/02.messageExtensions.a.searchCommand/appsettings.Development.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+	"Logging": {
+		"LogLevel": {
+			"Default": "Information",
+			"Microsoft": "Warning",
```
Comment (singhk97) [inline dotnet/samples/02.messageExtensions.a.searchCommand/appsettings.Development.json:5]: Should change this to "Information" as well

**Review by singhk97 — APPROVED** [review]

**Pull request merged**